bench/COMPUTE_UNITS.md

```diff
@@ -16,95 +16,95 @@ The programs and their tests are located in [/tests/bench](https://github.com/co
-| boxedAccountSized1          | 917           | -   |
-| boxedAccountSizedInit2      | 10828         | -   |
-| boxedAccountSized2          | 1463          | -   |
-| boxedAccountSizedInit4      | 19703         | -   |
-| boxedAccountSized4          | 2543          | -   |
-| boxedAccountSizedInit8      | 37919         | -   |
-| boxedAccountSized8          | 4898          | -   |
-| boxedAccountUnsizedInit1    | 6240          | -   |
-| boxedAccountUnsized1        | 972           | -   |
-| boxedAccountUnsizedInit2    | 11048         | -   |
-| boxedAccountUnsized2        | 1570          | -   |
-| boxedAccountUnsizedInit4    | 20138         | -   |
-| boxedAccountUnsized4        | 2768          | -   |
-| boxedAccountUnsizedInit8    | 38791         | -   |
-| boxedAccountUnsized8        | 5347          | -   |
-| boxedInterfaceAccountMint1  | 2296          | -   |
-| boxedInterfaceAccountMint2  | 4129          | -   |
-| boxedInterfaceAccountMint4  | 7783          | -   |
-| boxedInterfaceAccountMint8  | 15281         | -   |
-| boxedInterfaceAccountToken1 | 2023          | -   |
-| boxedInterfaceAccountToken2 | 3582          | -   |
-| boxedInterfaceAccountToken4 | 6692          | -   |
-| boxedInterfaceAccountToken8 | 13098         | -   |
-| interfaceAccountMint1       | 2364          | -   |
-| interfaceAccountMint2       | 5030          | -   |
-| interfaceAccountMint4       | 9803          | -   |
-| interfaceAccountMint8       | 18400         | -   |
-| interfaceAccountToken1      | 2091          | -   |
-| interfaceAccountToken2      | 3948          | -   |
-| interfaceAccountToken4      | 7547          | -   |
-| interface1                  | 1059          | -   |
-| interface2                  | 1479          | -   |
-| interface4                  | 1900          | -   |
-| interface8                  | 3646          | -   |
-| program1                    | 1053          | -   |
-| program2                    | 1467          | -   |
-| program4                    | 1878          | -   |
-| program8                    | 3598          | -   |
-| signer1                     | 1018          | -   |
-| signer2                     | 1484          | -   |
-| signer4                     | 1984          | -   |
-| signer8                     | 3880          | -   |
-| systemAccount1              | 1072          | -   |
-| systemAccount2              | 1590          | -   |
-| systemAccount4              | 2195          | -   |
-| systemAccount8              | 4305          | -   |
-| uncheckedAccount1           | 1014          | -   |
-| uncheckedAccount2           | 1475          | -   |
-| uncheckedAccount4           | 1965          | -   |
-| uncheckedAccount8           | 3841          | -   |
+| Instruction                 | Compute Units | +/-            |
+| --------------------------- | ------------- | -------------- |
+| accountInfo1                | 810           | 🟢 **-20.20%** |
```

Comment:
It's interesting that a single account instruction saves more compute units in both absolute and relative terms than multiple accounts instructions with the new implementation.